MAUI/ImageEditor/zooming.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: Zooming in .NET MAUI ImageEditor Control | Syncfusion
+description: Learn here all about how to Zooming in Syncfusion .NET MAUI ImageEditor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Zooming in .NET MAUI ImageEditor (SfImageEditor)
+
+The image editor control in .NET MAUI provides support for zooming and panning actions on an image.
+
+The following properties are used for zooming feature of the image editor control:
```

Comment:
remove this
The following properties are used for zooming feature of the image editor control:

* EnableZooming
* ZoomLevl
* Maximum ZoomLevel

Reply:
Removed